skycoin/skywire
# Fix/false positive app err logs

Did you run `make format && make check`?
yes

Fixes #1264

 Changes:	
- Fixed false positive stderr logs 
- Added errors
- Fixed routing table logs
- Added app started and stopped info logs

How to test this PR:
1. Start visor without root `./skywire-visor -c skywire-config.json`
2. Start and stop all apps
3. Check logs
4. Start visor root `sudo ./skywire-visor -c skywire-config.json`
5. Start and stop all apps 
6. Check logs
7. Try this on linux, mac and windows (use administrator on win)
